invenio_communities/assets/semantic-ui/js/invenio_communities/community/communities_items/ComputerTabletCommunitiesItem.js

```diff
@@ -20,7 +20,7 @@ export const ComputerTabletCommunitiesItem = ({ result }) => {
     : i18next.t("Restricted");
   const visibilityIcon = isPublic ? undefined : "ban";
   return (
-    <Item key={result.id} className="computer tablet only flex">
+    <Item key={result.id} className="computer tablet only flex community-item">
```

Comment:
This seems to be implemented in`app-rdm` and `invenio-communities`
However, I fixed it in both places.